.travis.yml

```diff
@@ -18,6 +18,8 @@ matrix:
   ## linters for json and JavaScript files
     - language: node_js
       python:
+      node_js:
+        - 7
```

Comment:
@cclauss we need to have node versioning, if not travis-ci will install 0.10 version, see the build after your PR #184
https://travis-ci.org/Fiware/dataModels/jobs/292149216